examples/crhmc_sampling/data/afiro.mm

```diff
@@ -0,0 +1,112 @@
+%%MatrixMarket matrix coordinate real general
```

Comment:
What does MatrixMarket mean?

Reply:
It is a popular sparse matrix file format: https://math.nist.gov/MatrixMarket/mmio-c.html

Reply:
Great, could you please add this sentence as a comment?